play/jaffle-shop-dbt/README.md

```diff
@@ -0,0 +1,147 @@
+## dbt's `jaffle_shop` + Materialize
+
+If you've used dbt, odds are that you've run across dbt's beloved [`jaffle_shop`] demo project.
+`jaffle_shop` allows users to quickly get up and running with dbt, using some spoofed, static
+data for a fictional [jaffle shop].
+
+At [Materialize], we specialize in maintaining fast and efficient views over your streaming data.
+While we work on hosting a public source of demo streaming data for analytics, we wanted to provide
+those familiar with dbt with an easy way to get up and running with our [`dbt-materialize`] adapter and
+`jaffle_shop`'s static data.
+
+> Note: This demo won't highlight what's powerful about Materialize. For that, check out our
+  `wikirecent-dbt` demo or our docs!
+
+[`jaffle_shop`]: https://github.com/fishtown-analytics/jaffle_shop
+[jaffle shop]: https://australianfoodtimeline.com.au/jaffle-craze/
+[Materialize]: https://materialize.com/
+[`dbt-materialize`]: https://pypi.org/project/dbt-materialize/
+[`wikirecent-dbt`]: https://github.com/MaterializeInc/materialize/blob/main/play/wikirecent-dbt/README.md
+
+## Setting up a `jaffle_shop` with Materialize
+
+The good news is that setting up the `jaffle_shop` project with Materialize as your data warehouse
+is easy. All you need to do is follow the [traditional `jaffle_shop`] steps with a few modifications:
+
+1. Following [the usual instructions], install dbt, clone the `jaffle_shop` repository, and navigate
+   to your cloned repo on your machine.
+
+1. In your cloned `dbt_project.yml`, make the following changes to the [model materializations]:
+    ```nofmt
+    models:
+        jaffle_shop:
+            marts:
+                core:
+                    materialized: materializedview
+                    intermediate:
+                        materialized: view
+            staging:
+                materialized: view
+                tags: ["staging", "hourly"]
+    ```
+    Only materializing your `core` business models as materialized views, without materializing
```

Comment:
Would it make sense to preface this with "Tip: "? It's too bad that GitHub's markdown doesn't have [admonitions](https://python-markdown.github.io/extensions/admonition/) to render things nicely. One can use emoji for this as well, but I haven't found any that I like for this purpose. Of course, feel free to use emoji if there is one that you like!

Reply:
Makes sense to me! I just prefaced it with "Tip: ", like you suggested, but it might be nice to come up with a universal emoji glossary or something for our docs.